apixu-weatherbot1.0.tcl

```diff
@@ -0,0 +1,498 @@
+#!usr/bin/env tclsh
+
+########################################################################################
+# Name              apixu-weatherbot1.0.tcl
+#
+# Author            Bruce Olivier <bolivierjr@gmail.com>
+#
+# Description       Uses the Apixu weather API to grab the data.
+#
+#                   Requires channel to be flagged with +weather-apixu
+#                       example: .chanset #chan +weather-apixu
+#
+#                   Commands:
+#                       .set 0-2 <location>
```

Comment:
0,1,2 was used in the old weather script. I figured the users might get grumpy if I change it lol.  It would make more sense though! That could save me a couple lines of code, actually.

Reply:
why not both? legacy support and clarity for new users.